php/php.editor/src/org/netbeans/modules/php/editor/actions/FixUsesAction.java

```diff
@@ -281,32 +282,104 @@ public static class Options {
         private final boolean preferGroupUses;
         private final boolean startUseWithNamespaceSeparator;
         private final boolean aliasesCapitalsOfNamespaces;
+        private final boolean putInPSR12Order;
         private final PhpVersion phpVersion;
 
+        public static class Builder {
```

Comment:
I've added the `Builder` because it's easy to understand when we set options with unit tests.